src/constants/brand.constants.ts

```diff
@@ -0,0 +1,20 @@
+export const brandConfig = {
+    brandName: "Deriv",
+    domainName: "Deriv.com",
+    legalEntities: {
+        fx: "Deriv (FX) Ltd",
+        maltainvest: "Deriv Investments (Europe) Limited",
+        svg: "Deriv (SVG) LLC",
+        v: "Deriv (V) Ltd",
+    },
+    platforms: {
+        trader: "Deriv Trader",
+        dbot: "Deriv Bot",
+        mt5: "Deriv MT5",
+        ctrader: "Deriv cTrader",
+        dxtrade: "Deriv X",
+        smarttrader: "SmartTrader",
+        bbot: "Binary Bot",
+        go: "Deriv GO",
+    },
+};
```

Comment:
```suggestion
} as const;
```

Reply:
It's important for our library to let consumers see all the types. If we don't do this, consumers won't know what's the **brandConfig** type